cvt.py

```diff
@@ -12,6 +12,7 @@ class Cvt(kp.Plugin):
     ITEMCAT_RELOAD_DEFS = kp.ItemCategory.USER_BASE + 2
     ITEMCAT_CREATE_CUSTOM_DEFS = kp.ItemCategory.USER_BASE + 3
 
+    CONFIG_SECTION_MAIN = "main"

```

Comment:
I do not see where are you using this new variable to load the label from the config file.

Reply:
Shuzo san, I am happy to see you find this extension useful. In general, other than the above comment, I don't see an issue (the utf-8 is good catch I did not notice).

I am wondering whether it would make more sense to have a pattern of cvtdefs[_{locale}].json where the optional _{locale} is automatically detected such that we can add cvtdefs_jp.json to the plugin and then when used in a Japanese computer, it will have the Japanese version.

Reply:
Can you run the following on your system and copy the result here:

```
>>> import locale
>>> locale.getdefaultlocale()
('en_US', 'cp1252')
```

Reply:
Mr.DrorHarari thanks for your review!
And so sorry for my late reply.

### CONFIG_SECTION_MAIN 

This used at l.106 on cvt.py to specify a section `[main]`.

### Run python code on my system.

Return is

```
>>> import locale
>>> locale.getdefaultlocale()
('ja_JP', 'cp932')
```

Yes, certainly `_jp.json` style may be more sensible.
Hmm, I am debating about.
In japan, shift-jis (cp932) is traditionally used on windows for a long time.
For example, Windows notepad used shift-jis as default until windows 10 May 2019 Update.
But now utf-8 is default. And utf-8 is world standard, encoding of cvtdefs.json is also utf-8.
So I think using `encoding = "utf-8"` option is more modern style and there are few problems perhaps.

Reply:
> sorry for

We're living in internet time - we pop up and go away on irregular schedules...

So if you are interested, you can push a PR that will try to open cvtdefs[-{locale}].json (in the case of Japan it will be cvtdefs-ja_JP.json) in the plugin or the user directory before trying to open it without the locale. Then I can package your Japan-specific file in the plugin for the benefit of other Japanese users. Your call.

As for UTF-8, it is clear that we don't want the json files to be encoded in shift-jis, big5 or other local encoding, UTF-8 is there for good reason. I just am working in pure ASCII environment so I did not notice the missing UTF-8 in the open...

Reply:
OK. Sounds good! Japan-specific cvtdefs file is useful. I try it.
And I add `data/cvtdefs-ja_jp.json` file with some japanese convert setting.
Wait a few weeks.

Reply:
Great. Take your time. Just to be sure, the idea is not to look specifically for ja_JP but to look for what locale.getdefaultlocale()[0] returns. It may be also a good idea to have a locale configuration item so if you are a Japanese person using a machine where the locale is not ja_JP you just need to set that parameter in the configuration file.

Reply:
Yes, I thought about it...
Hmm... I suggest this architecture.

* Add optional list setting `def_files` in plugin configuration.

* Default: load cvtdefs.json only.

```
[main]
# def_files =          // Commented out. No need setting to use cvtdefs.json.
```

* If user need other def files.

```
[main]
def_files = cvtdefs.json        // It is default general defines. User can overwrite.
            cvtdefs_jp.json     // It has some japanese specific defines. User can overwrite too.
            ... and more, for example each countries, each jobs. 
```

In plugin, load all def files and combine.
I think this can be loosely coupled from python behavior.
How about it?

Reply:
Shuzo san, I decided to make the change myself. I took the opportunity to simplify the cvtdefs.json so that you don't need to copy it in whole. You can put cvtdefs-ja_JP.json in the user folder and all the measures in it would get added to the common ones. If you want me to package it in the plugin, send a PR. 

Examples can be seen in the Cvt.ini and in the repository readme.  /d

Reply:
By the way, I have removed the code that chagnes the Cvt: label because (1) it did not work and (2) I am not sure what is the need for that.

Reply:
OK. I understand.